README.md

```diff
@@ -185,11 +185,10 @@ To publish locally, you'll need to do the following...
 ####Sign In
 
 To publish locally, you'll need an account.
-* Go to [http://localhost:3000/account](http://localhost:3000/account)
-* Click ``Join Webmaker`` and complete the process, you can use a fake email
-* When you've created your account, click ``Set permanent password instead``
-  * This lets you authenticate your account without needing email
-* Go back to Thimble and Log In with your new account
+* Go to [http://localhost:3000](http://localhost:3000)
```

Comment:
@gideonthomas, @ryanwarsaw do we want them to do this directly (e.g., go to login) or do we want to tell them to do it via Thimble?  I think the latter, no?